packages/orm/src/EntityManager.ts

```diff
@@ -196,17 +195,17 @@ export class EntityManager<C = unknown> {
    * of our `flush` loop, and b) cascade deletions before we recalc fields & run user hooks,
    * so that both see the most accurate state.
    */
-  private pendingCascadeDeletes: Entity[] = [];
-  public dataloaders: Record<string, LoaderCache> = {};
-  // This is attempting to be internal/module private
-  __data = {
-    joinRows: {} as Record<string, JoinRow[]>,
-    /** Stores any `source -> downstream` reactions to recalc during `em.flush`. */
-    rm: new ReactionsManager(),
-  };
-  private hooks: Record<EntityManagerHook, HookFn[]> = {
-    beforeTransaction: [],
-    afterTransaction: [],
+  #pendingCascadeDeletes: Entity[] = [];
+  #dataloaders: Record<string, LoaderCache> = {};
+  #joinRows: Record<string, JoinRow[]> = {};
+  /** Stores any `source -> downstream` reactions to recalc during `em.flush`. */
+  #rm = new ReactionsManager();
+  #hooks: Record<EntityManagerHook, HookFn[]> = { beforeTransaction: [], afterTransaction: [] };
+  private __api: EntityManagerInternalApi = {
```

Comment:
This `__api` stays TS private / not-`#` so that our `getEmInternalApi` can grab it.